jet/hazelcast-3-connector/hazelcast-3-connector-jobs/src/main/java/com/hazelcast/samples/jet/hz3connector/Trade.java

```diff
@@ -0,0 +1,56 @@
+package com.hazelcast.samples.jet.hz3connector;
+
+import javax.annotation.Nonnull;
+import java.io.Serializable;
+import java.util.Objects;
+
+public class Trade implements Serializable {
+
```

Comment:
Could we include serialVersionUID since it is recommended per official docs - https://docs.hazelcast.com/hazelcast/5.0/serialization/implementing-java-serializable